e2e-tests/tests/conftest.py

```diff
@@ -519,3 +505,57 @@ def _wait_until(condition, *args, timeout=10, poll_interval=10):
         raise TimeoutError(f"WAIT UNTIL function TIMED OUT after {timeout}s on {condition} with args {args}.")
 
     yield _wait_until
+
+
+@fixture(scope="session")
+def write_file():
+    saved_files = {}
+
+    def _write_file(runner: Runner, content: str):
+        filepath = f"/tmp/{uuid.uuid4().hex}"
+        content_json = json.dumps(content)
+        runner.run(f"echo '{content_json}' > {filepath}")
+
+        if runner not in saved_files:
+            saved_files[runner] = []
+        saved_files[runner].append(filepath)
+        return filepath
+
+    yield _write_file
+
+    for runner, filepaths in saved_files.items():
+        logging.info("Cleaning up temporary cli files on remote host...")
+        cmd = f"rm {' '.join(filepaths)}"
+        runner.run(cmd)
+
+
+@fixture(scope="session")
+def governance_skey_with_cli(config: ApiConfig):
+    """
+    Securely copy the governance authority's init skey (a secret key used by the PCSC CLI to authorize admin operations)
+    to a temporary directory on the remote machine and update the path in the configuration. The temporary directory is
+    deleted after the test completes.
+
+    This fixture is executed only if SSH is configured in the stack settings, implying that the PCSC CLI (which
```

Comment:
Is "PCSC CLI" concept present in other areas of the code or is it obsolete because the node has all the CLIs now?

Reply:
I'll replace it with `smart-contracts`